SharedProjects/Babylon2GLTF/GLTFExporter.Animation.cs

```diff
@@ -308,28 +226,10 @@ private void ExportBoneAnimation(GLTFAnimation gltfAnimation, int startFrame, in
 
             if (babylonBone.animation != null && babylonBone.animation.property == "_matrix")
             {
-                RaiseMessage("GLTFExporter.Animation | Export animation of bone named: " + babylonBone.name, 2);
+                logger.RaiseMessage("GLTFExporter.Animation | Export animation of bone named: " + babylonBone.name, 2);
 
                 var babylonAnimation = babylonBone.animation;
 
-                // Optimize animation
```

Comment:
I removed this optimization pass because we should already do it when exporting babylon. we should minimize our changes to the input Babylon file and just convert what we have to GLTF